tutorials/simulation/plot_dics.py

```diff
@@ -221,10 +222,11 @@ def coh_signal_gen():
 brain.add_foci(vertices[1][0], coords_as_verts=True, hemi='rh')
 
 # Rotate the view and add a title.
-fig = brain._figures[0][0]
-mne.viz.set_3d_view(fig, azimuth=0, elevation=0, distance=550,
-                    focalpoint=[0, 0, 0])
-mne.viz.set_3d_title(fig, 'MNE-dSPM inverse (RMS)')
+with use_3d_backend("mayavi"):
+    fig = brain._figures[0][0]
+    mne.viz.set_3d_view(fig, azimuth=0, elevation=0, distance=550,
+                        focalpoint=[0, 0, 0])
+    mne.viz.set_3d_title(fig, 'MNE-dSPM inverse (RMS)')
```

Comment:
Instead just do brain.show_view https://pysurfer.github.io/generated/surfer.Brain.html#surfer.Brain.show_view

And maybe in `stc.plot` you could use `time_label='MNE-dSPM inverse (RMS)'` to set the title instead?

Reply:
I had some trouble to make it work with `time_label` locally. The window's title wasn't updated for some reason and I obtained the "Mayavi Scene" instead. Maybe I did something wrong, I'll push my changes.